animal_crossing.py

```diff
@@ -110,9 +196,11 @@ def _stalk_list_sale_prices(cmd):
 	current_time = datetime.datetime.now(datetime.timezone.utc)
 	sunday = _date_to_sunday(current_time)
 	cur = db.execute('''
-	SELECT username, week_index, expiration, price FROM sell_price
-	JOIN user ON sell_price.user_id = user.id
-	WHERE week_local == ?
+	SELECT username, week_index, sp.expiration, bp.price AS buy_price, sp.price
+	FROM sell_price sp
+	JOIN buy_price bp ON bp.user_id = sp.user_id AND bp.week_local = sp.week_local
```

Comment:
```suggestion
	LEFT JOIN buy_price bp ON bp.user_id = sp.user_id AND bp.week_local = sp.week_local
```
tried this out with `test.py` and this (INNER) JOIN excludes people with no buys